mii/batching/ragged_batching.py

```diff
@@ -721,6 +719,44 @@ def flush(self, uids: List[int]) -> None:
             self.inference_engine.flush(uid)
 
 
+@dataclass
+class StreamState:
+    prev_tok: str
+    tids: List[int]
```

Comment:
This might be easily confused with the thread ID variable `tid`.  Could we use `token_ids` or replace `tid` with `thread_id` in the `get_response` method?